base_station/gui/src/components/Amino.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+<div class="wrap">

```

Comment:
also interesting that science changes are in here

Reply:
ack